app/services/google_service/client_service.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module GoogleService
```

Comment:
Can we remove `Service` from namespace, and only use `Google:ClientService`.

Reply:
Yes we can :) 
That will solve my question "should I pluralize GoogleService[s] or not?" :)